src/celeritas/ext/GeantImporter.cc

```diff
@@ -404,7 +470,10 @@ ImportData::ImportOpticalMap import_optical()
     auto num_materials = pct.GetTableSize();
     CELER_ASSERT(num_materials > 0);
 
+    auto const particle_map = optical_particles_map();
```

Comment:
```suggestion
    auto const& particle_map = optical_particles_map();
```
You're making a copy again :)